vizro-core/tests/e2e/vizro/dashboards/default/dashboard.py

```diff
@@ -74,6 +87,15 @@
         collapsible_containers_grid,
         collapsible_containers_flex,
         vizro_url_and_download_page,
+        set_control_graph_interactions_page,
+        set_control_ag_grid_interactions_page,
+        drill_through_filter_page_1,
+        drill_through_filter_page_2,
+        drill_through_parameter_page_1,
+        drill_through_parameter_page_2,
+        drill_through_filter_ag_grid_page_1,
+        drill_through_filter_ag_grid_page_2,
+        set_control_drill_down_page,
```

Comment:
Would you mind renaming some stuff in this PR? So, replace the "interaction" with the "cross_filter"  in the filename, variable/function names and comments. Also, would you align the "set_control" prefix by removing it everywhere or adding it everywhere? 

```suggestion
        cross_filter_graph_page,
        cross_filter_ag_grid_page,
        drill_down_graph_page,
        drill_through_filter_graph_source_page,
        drill_through_filter_graph_target_page,
        drill_through_parameter_graph_source_page,
        drill_through_parameter_graph_target_page,
        drill_through_filter_ag_grid_source_page,
        drill_through_filter_ag_grid_target_page,
```

Reply:
Thanks! Changed. Let's leave all this combinations for now.